omod/src/test/resources/logEntriesTestDataset.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<dataset>
+    <idgen_log_entry id="1" source="Generator" generatedby="Admin" identifier="100892" comment="New patient " dateGenerated="3/10/2016"/>
```

Comment:
Does this dataset load? I though generated by takes an integer.

Reply:
The same applies for "source"

Reply:
Not sure. Generatedby is of type User. Are users identified by integers?

Reply:
Have you tried to look at existing test datasets that take the same type?

Reply:
I assume _user_ is similar to _creator_ in [customActiveListTest](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-1.8/src/test/resources/customActiveListTest.xml). Is that so?

Reply:
You gat it! 😊